content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -285,8 +285,8 @@ If you restrict `list` or `watch` by resourceName, clients must include a `metad
 For example, `kubectl get configmaps --field-selector=metadata.name=my-configmap`
 {{< /note >}}
 
-Rather than referring to individual `resources` and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
-For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `apiGroups` and `resourceNames` an empty set means that everything is allowed.
+Rather than referring to individual `resources` 、`apiGroups`and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
+For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `resourceNames` an empty set means that everything is allowed. `apiGroups` is accessible only to resources that do not have apiGroups if it is an empty set.
```

Comment:
I would split this paragraph... mixing the non-resource information and resource information is confusing.

```
Rather than referring to individual `resources` 、`apiGroups`and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
For `resourceNames` an empty set means that everything is allowed.

For `nonResourceURLs`, you can use the wildcard `*` symbol as a suffix glob match.
```

Reply:
@zmquan , please take a look at the above comment about splitting the paragraph

Reply:
It has been adjusted according to your suggestion. Please have a look  @liggitt 
@reylejano  cc